staff/gustavo-rojas/git/README.md

```diff
@@ -0,0 +1,42 @@
+
+#To create a repository in the actual directory
+git init
+
+# To see what files status are
+git status
+
+#To add files to the stagging area
+git add [The name of the file]
+# To add all the files with updates
```

Comment:
Keep consistency, if you are going to leave one line after each command, do the same for all.